### PR TITLE
Changing CLI Output Parameter

### PR DESCRIPTION
**Pull request by emilyyzhangg**
Changing so that GoldPolish-Target command accepts an output file name and not a prefix name

### CODE REVIEW

File: scripts/goldpolish-target.py [hunk 1]
```diff
@@ -129,6 +129,12 @@ def get_mapping_info(minimap2, ntLink):
         raise ValueError("Unknown mapping tool")
     return (mapper, s_goldpolish)
 
+def split_output_file_name(output_file):
+    """Removes .fa from end of output file to create a prefix"""
+    if output_file.endswith('.fa'):
```
Comment (lcoombe) [inline scripts/goldpolish-target.py:134]: To be safe, I think we should also handle other conventional fasta suffices? (.fasta, .fna)

**Review by lcoombe — COMMENTED** [review]
Thanks Emily! Just one small suggestion.

File: scripts/goldpolish-target.py (hunk 1 above)
Comment (emilyyzhangg) [reply scripts/goldpolish-target.py:134]: Good catch!

**Review by lcoombe — APPROVED** [review]
Thanks Emily!

**Pull request merged**